ninep/tmpfs/archive.go

```diff
@@ -306,8 +322,47 @@ func ReadImageCpio(r io.ReaderAt) (*Archive, error) {
 				return nil, err
 			}
 		}
+		Debug("Add record for %q: %v", record.Name, file)
+		files[record.Name] = file
+	}
+
+	for _, record := range records {
+		Debug("%v: check %#o is symlink %#o", record, record.Mode, cpio.S_IFLNK)
+		if record.Mode&0170000 != cpio.S_IFLNK {
```

Comment:
Why the octal comparison?  Wouldn't 0xf000 be more readable?

Reply:
Mainly because that constant is defined -- everywhere -- in octal. Including in the Go os package.

Reply:
so if you change that constant, it looks unlike every time it's been used for the past 50 years. You want to think hard before you do that. grep -r on /usr/include on almost anything and you can see what I mean.

Reply:
ok, octal it is then